corehq/apps/translations/tasks.py

```diff
@@ -152,3 +153,31 @@ def backup_project_from_transifex(domain, data, email):
         filename = "%s-TransifexBackup.zip" % project_details.get('name')
         email.attach(filename=filename, content=tmp.read())
         email.send()
+
+
+@task
+def download_project_from_hq(domain, data, email):
```

Comment:
this it should probably be changed to `email_x`

Reply:
+1

Reply:
PR #24451 ... kinda. Scope of what should be renamed will need a slightly broader conversation, but I agree with the principle. "Download" is inaccurate and misleading.